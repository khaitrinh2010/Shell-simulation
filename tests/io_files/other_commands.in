cat students.csv 
grep "MATH" students.csv
sort students.csv
cut -d, -f2 students.csv

