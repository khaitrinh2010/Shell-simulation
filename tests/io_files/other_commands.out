>> Code,Title,Semester,Credits
INFO1001,Introduction to Programming,4
INFO2112,Software Development,5
INFO1112,Data Structures and Algorithms,2
INFO3001,Advanced Topics in Information Systems,1
INFO2005, Football,7
INFO2001,Database Systems,10
MATH1064, Discrete Math,6  
ELEC1601, Computer System,3
>> MATH1064, Discrete Math,6  
>> Code,Title,Semester,Credits
ELEC1601, Computer System,3
INFO1001,Introduction to Programming,4
INFO1112,Data Structures and Algorithms,2
INFO2001,Database Systems,10
INFO2005, Football,7
INFO2112,Software Development,5
INFO3001,Advanced Topics in Information Systems,1
MATH1064, Discrete Math,6  
>> Title
Introduction to Programming
Software Development
Data Structures and Algorithms
Advanced Topics in Information Systems
 Football
Database Systems
 Discrete Math
 Computer System
>> >> 
